framework/include/base/Moose.h

```diff
@@ -32,8 +32,12 @@ class MooseEnumItem;
 class ExecFlagEnum;
 class MooseVariableFEBase;
 
+#ifdef LIBMESH_HAVE_PETSC
```

Comment:
Can we just add this to MOOSE already?
```
#ifndef LIBMESH_HAVE_PETSC
   YOUR DUMB
#endif 
```

Reply:
I agree...

Reply:
I agree...

Reply:
Wow, it appears you doubly agree! Is that message PC enough?

Reply:
Stupid github slowness. Wow this is the most constructive review I've ever had 😆